### DISCUSSION TITLE
AK 47 FULL AUTO RECONVERSION

### DISCUSSION THREAD

**Discussion by Max-programming-novice** [CleverRaven/Cataclysm-DDA#48970] [Ideas]
As i know, in real life, (thanks Brandon Herrera and some other youtubers) semi-auto Ak 47 can be reconversed into full auto one... So... Knowing that cataclysm has Ar15 reconversion, and AKM (that easily kills zombie hulks), maybe devs should add full auto Ak47 and Mini drako (and... maybe Saiga12/410)... But it should be a bit less reliable than AKM... I guess that this craft will be pretty popular (since you can find Ak47 at every 5th farm or Potters cottage or at every 3rd gunshop, that you look)... So... I guess you can find this receipt at Fn 23-16 Marksmanship manual (like Ar15), or discover it yourself having 4 rifles or marksmanship skill..
What do you think about this idea...
And sorry, i login into Gitchub today, idk how this system works...

**Comment by actual-nh** [comment]
You might want to put it in as an issue - select "Issues" and then click on "New Issue". This would be a "Feature Request", I think; be sure to take a careful look at the template you're given.

**Reply by Max-programming-novice** [reply]
Thank you

**Comment by Max-programming-novice** [comment]
Ok... But i guess such receipts are a bit illegal...
P.s how you find receipts of Ar15 full auto reconversion mods?

сб, 29 мая 2021 г., 23:18 faefux ***@***.***>:

**Reply by actual-nh** [reply]
Hrm. Likely depends on what country one is in. AFAIK, in the US for instance, it's perfectly legal to publish such (although there might be some court fights and some ISP takedowns to avoid controversy); doing said conversion, including for making a video, would be another matter entirely.
